README.md

```diff
@@ -80,6 +80,28 @@ withObservables(triggerProps, getObservables)
 //   }))
 ```
 
+## Typescript
+
+The TypeScript bindings expose a helper type, `ObservablifyProps<Props, ObservableKeys, ObservableConvertibleKeys>` which can make it easier to wrap components without duplicating interfaces:
```

Comment:
oh, so TypeScript can't figure those things out by itself automagically? :/

can you send a quick PR updating https://github.com/Nozbe/WatermelonDB/blob/master/CHANGELOG.md with a link to withObservable repo changelog? Otherwise I'm afraid no one will figure out that it has types (no one looks at _this_ repo)

Reply:
Yeah, the definition seems to become a bit circular without this - it isn't always necessary, just when there are other props floating around. Will do the WatermelonDB PR now!

Reply:
> If I have time I'm going to have a go at fixing the TS subclassing issues that project is having too.

Awesome! I don't use TS, so it's only thanks to the community like you that the types exist and are improved over time :)